config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  get 'static_page/home'
+  get 'static_page/help'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
 end
```

Comment:
use double quotes instead single quotes in rb, erb files. use single quotes instead double quotes in the js file. Remove all comment

Reply:
edited